apps/website/src/components/forms/EntryForm.tsx

```diff
@@ -80,7 +80,7 @@ export const EntryForm: React.FC<{
           <Heading>{form.label}</Heading>
         </header>
 
-        {config.intro && <Markdown content={config.intro} />}
+        {config.intro && <Markdown content={config.intro} dark={true} />}
```

Comment:
```suggestion
        {config.intro && <Markdown content={config.intro} dark />}
```
`={true}` on props is redundant